### PR TITLE
ta - user can view index of link images on landing page

### LINKED ISSUE #3: Image Index
#### As a user I want the homepage to display all saved images.

__Story__:

Now that we are saving images for our users, we want them to be able to view
the list of images in the system. You might be thinking, "what if my users only
want to view images of cats", or "what if my users only want to see the images
they uploaded". Those are great questions. Maybe you will have the opportunity
to add those features in a future story. For now, we will keep it simple and
quickly deliver a little more value to our users.

__Acceptance criteria__:
- [x] New images that are added show up on the homepage.
- [x] These images are persisted if the browser is closed or even if the
  server is restarted.
- [x] Images are not displayed wider than 400px.
- [x] Newest images appear first.

__Dependencies__:
- Save Image Link

### PR DESCRIPTION
**Pull request by treyamador**
Closes #3

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
-    This website is under construction.  You can upload images if you'd like.
-    Come visit us again sometime to see some images from your friends share with your friends.
+    Browse the images uploaded by other users below.
   </p>
+
+  <div class="row">
+    <% @images.each do |img| %>
+      <div class="col-xs-18 col-6 col-md-3">
+        <%= link_to img do %>
+          <div class="thumbnail">
```
Comment (dshen94) [inline app/views/pages/home.html.erb]: I noticed that one of the issue's requirements was that the images not be displayed wider than 400px, I wasn't been able to find that anywhere. Were you able to add that anywhere i.e was that a part of the built-in thumbnail class or via the columns?

Comment (treyamador) [reply app/views/pages/home.html.erb]: Yes, that's in app/assets/stylesheets/images.scss.  I had added it in a previous commit.

Comment (treyamador) [reply app/views/pages/home.html.erb]: Sorry, I'm mistaken.  I had a size limit previously, but removed it since the thumbnail takes care of it.

Comment (dshen94) [reply app/views/pages/home.html.erb]: Sounds good, it looks that way on the deployment as well. 👍

**Pull request merged**